src/common/styles-dictionary/tailwindFormat.js

```diff
@@ -0,0 +1,101 @@
+/* eslint-disable no-restricted-syntax */
+/* eslint-disable @typescript-eslint/no-var-requires */
+/* eslint-disable import/no-extraneous-dependencies */
+
+const StyleDictionary = require("style-dictionary");
+const cssFont = require("css-font");
+
+StyleDictionary.registerFormat({
+  formatter({ dictionary, options }) {
+    function getName(name) {
+      const joinedName = typeof name === "string" ? name : name.join("-");
+      const mappedName = options.remapNames?.[joinedName] ?? joinedName;
+      const result = options.prefix
+        ? `${options.prefix}-${mappedName}`
+        : mappedName;
+
+      return result.toLowerCase().replace(" ", "-");
+    }
+
+    function transformDictionary(tokens, keys) {
+      const result = {};
+
+      for (const [name, token] of Object.entries(tokens)) {
+        if (!keys || keys.includes(name)) {
+          result[getName(name)] = token.value;
+        }
+      }
+
+      return result;
+    }
+
+    function transformColor(tokens) {
+      const colors = {};
+
+      for (const [name, colorTokens] of Object.entries(tokens)) {
```

Comment:
what is the difference between colorToken and token? is it possible to rename them to something more english-y?